### PR TITLE
Fix: Adjust apply button size (fixes #543)

### LINKED ISSUE #543: The custom date range picker apply button is too small
**Describe the bug**
The custom date range picker's apply button is not sized correctly

**To Reproduce**
1. Go to https://yahoo.github.io/navi/#/reports/1/view
2. Click on Last 7 days
3. Click on Custom range in the dropdown
4. The apply button is too small

**Expected behavior**
Should be the same size as the Reset button

**Screenshots**
Screen Shot 2019-10-03 at 3 01 24 PM

**Desktop:**
 - OS: macOS
 - Browser: Google Chrome
 - Version: 77

### PR DESCRIPTION
**Pull request by duncansanderson**
Resolves #543 

## Description
The custom date range picker's apply button is not sized correctly

## Proposed Changes
Change the apply button to match the reset button

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by mattgraham1995** [comment]
Can you post a screenshot?

**Comment by kevinhinterlong** [comment]
Ran it locally, here's the screenshot
Screen Shot 2019-10-03 at 4 34 25 PM

File: packages/reports/app/styles/navi-reports/components/navi-date-range-picker.less [hunk 1]
```diff
@@ -55,10 +55,9 @@
 
   &__apply-btn {
     font-size: @font-size-mid;
-    border-radius: 5px;
     border-width: 0 0 3px 0;
-    padding: 2px 13px;
-    margin: 0 0 4px 10px;
+    margin: 0 0 0 10px;
```
Comment (mattgraham1995) [inline packages/reports/app/styles/navi-reports/components/navi-date-range-picker.less]: `margin-left: 10px;` ?

Comment (daniellevinson) [reply packages/reports/app/styles/navi-reports/components/navi-date-range-picker.less]: @mattgraham1995 maybe with `margin: 0` to not unset the other margins

Comment (mattgraham1995) [reply packages/reports/app/styles/navi-reports/components/navi-date-range-picker.less]: Yeah I also wondered that, but we checked in the existing version and there is no conflicting margin.

**Pull request closed without merging**